e3sm_diags/plot/aerosol_aeronet_plot.py

```diff
@@ -108,7 +108,4 @@ def plot(
 
     plt.loglog(ref_site_arr, test_site_arr, "kx", markersize=3.0, mfc="none")
 
-    # Configure legend.
-    plt.legend(frameon=False, prop={"size": 5})
-
```

Comment:
Removed unused `plt.legend()` that produces a UserWarning because there is no label for the data.